reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -40,7 +40,7 @@
     "id" : "GPL-3.0-only WITH GCC-exception-3.1 AND CC-BY-NC-3.0"
   }, {
     "_id" : 6,
-    "id" : "GPL-3.0-only"
+    "id" : "GPL-3.0-only WITH GCC-exception-3.1"
```

Comment:
Commit message: First sentence should end with "...handling of detected licenses", right?
Nit: "pior" -> "prior"

Reply:
Done.